sdk/cosmos/azure-cosmos/azure/cosmos/_execution_context/base_execution_context.py

```diff
@@ -120,7 +119,14 @@ def _fetch_items_helper_no_retries(self, fetch_function):
                 self._has_started = True
             new_options = copy.deepcopy(self._options)
             new_options["continuation"] = self._continuation
-            (fetched_items, response_headers) = fetch_function(new_options)
+            try:
+                (fetched_items, response_headers) = fetch_function(new_options)
+            except exceptions.CosmosHttpResponseError as e:
```

Comment:
question, if we do not capture and raise the exception specifically, then the exception should automatically bubble up to up layer?

Reply:
seems like you're right, there's no need to capture the exception at this point :)